src/app/components/Sidebar/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<Navigation /> should match snapshot 1`] = `
-.c12 {
+.c10 {
```

Comment:
Is this because the UI changed?

Reply:
yeah, these always change when hierarchy of components changes